src/screens/Assets/WalletView.js

```diff
@@ -436,20 +435,30 @@ class WalletView extends React.Component<Props, State> {
         }
         getRef={(ref) => { this.scrollViewRef = ref; }}
       >
+        { showDeploySmartWallet && (
+            smartWalletState.upgradeDismissed ? (
+              <SWActivationCard
+                message="To start sending and exchanging assets you need to activate Smart Wallet"
+                buttonTitle="Activate Smart Wallet"
+              />
+            ) : (
+              <InsightWithButton
+                title="Why Smart Wallet knocks out your old private key wallet?"
+                itemsList={initialSWInsights}
+                buttonTitle="Wow, that's cool"
+                onButtonPress={() => {
+                dismissSmartWalletUpgrade();
+              }}
+              />
+            ))
+        }
         <Insight
           isVisible={isInsightVisible}
           title={insightsTitle}
           insightChecklist={insightList}
           onClose={() => { hideInsight(); }}
           wrapperStyle={{ borderBottomWidth: 1, borderBottomColor: colors.border }}
         />
-        {blockAssetsView &&
-        <DeploymentView
-          message={deploymentData.error ? getDeployErrorMessage(deploymentData.error) : sendingBlockedMessage}
-          buttonAction={deploymentData.error ? () => deploySmartWallet() : null}
-          buttonLabel="Retry"
```

Comment:
That component was helping users whose deployment failed. What should they do now?